geopaparazzispatialitelibrary/src/eu/geopaparazzi/spatialite/database/spatial/core/daos/DaoSpatialite.java

```diff
@@ -436,9 +439,51 @@ public static void updateFeatureGeometry(String id, Geometry geometry, String ge
         sbIn.append(SpatialiteUtilities.SPATIALTABLE_ID_FIELD).append("=");
         sbIn.append(id);
         String insertQuery = sbIn.toString();
-        database.exec(insertQuery, null);
+        dbSpatialite.exec(insertQuery, null);
+    }
+    public static double[] longLat2Srid(Database dbSpatialite, double lon, double lat, String s_srid)  throws Exception {
```

Comment:
2015-09-19 14:09 GMT+02:00 Andrea Antonello <notifications@github.com>:

> In
> geopaparazzispatialitelibrary/src/eu/geopaparazzi/spatialite/database/spatial/core/daos/DaoSpatialite.java
> <https://github.com/geopaparazzi/geopaparazzi/pull/287#discussion_r39919038>
> :
>
> > @@ -436,9 +439,51 @@ public static void updateFeatureGeometry(String id, Geometry geometry, String ge
> >          sbIn.append(SpatialiteUtilities.SPATIALTABLE_ID_FIELD).append("=");
> >          sbIn.append(id);
> >          String insertQuery = sbIn.toString();
> > -        database.exec(insertQuery, null);
> > +        dbSpatialite.exec(insertQuery, null);
> > +    }
> > +    public static double[] longLat2Srid(Database dbSpatialite, double lon, double lat, String s_srid)  throws Exception {
>
> Why is this method necessary? A javadoc would help
>
Yes, is missing,
* is to transform the screen position to the srid of the map

for AbstractSpatialTable:

    /*
     * Convert a longitude and latitude to the table's
     * original SRID.
     */
    public abstract double[] longLat2Srid(double lon, double lat);

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geopaparazzi/geopaparazzi/pull/287/files#r39919038>.
>